src/pubsub/mod_pubsub.erl

```diff
@@ -61,6 +61,7 @@
 
 -define(STDTREE, <<"tree">>).
 -define(STDNODE, <<"flat">>).
+-define(STDNODE_MODULE, node_flat).
```

Comment:
How about `STDNODE_MODULE` rename to `STDNODE_DEFAULTMODULE` ??

Reply:
That's not a bad idea :)